assets/ts/modules/filterlist.ts

```diff
@@ -0,0 +1,46 @@
+// @ts-nocheck
+function filterlist(list: Element, input:Element) {
+
+  addFilterlistEventListeners(list,input);
+}
+
+function addFilterlistEventListeners(list: Element, input:Element) {
+
+  var timer;
+
+  input.addEventListener('keyup', (event) => {
+
+    clearTimeout(timer);
+      timer = setTimeout(function(){
+        filterTheList(list,input.value);
+      }, 1000);
```

Comment:
Would be nice if this was configurable I think. Or if you disagree, I think it feels sluggish to the point it feels like the app is responding slowly, 500ms might be better

Reply:
@arthurrigley I have just changed to 500, I think making it configurable would be overkill.